### PR TITLE
Add tracking

### LINKED ISSUE #339: Implement Matomo for usage analytics
**Reason (Why?)**
To improve the website a usage analytics tool should be implemented.

**Solution (What?)**
Use [Matomo](https://github.com/matomo-org/matomo) for usage analytics. 
Weigh up how many information to use for the analysis. Implement a Cookie consent if needed. 

**Relation to other Issues**
/

**Acceptance criteria**
/

### PR DESCRIPTION
**Pull request by theErikss0n**
This implements the usage of analytics in the client.
[Vue-Matomo](https://github.com/AmazingDreams/vue-matomo) is used in order to track site-urls and send those to our matomo server.
I extended our privacy notice with a declaration about matomo. The text is copied from OpenArtBrowser.

In order to only track in production and not during development or in staging, [vue modes](https://cli.vuejs.org/guide/mode-and-env.html#modes) and mode-specific env files are used.

@KaiDrieHda or @DianaPankratz can you please double check whether the privacy notice is all we need or if we need something else?

closes #339

### CODE REVIEW

**Review by xcaverxiii — APPROVED** [review]
looks ok

**Comment by theErikss0n** [comment]
Together with @DianaPankratz we found out, that AdBlockers are blocking our opt-out i-frame provided by matomo.
I'm working on an alternative.
This should not be merged until this is resolved.

**Review by DianaPankratz — CHANGES_REQUESTED** [review]
Please change the info text for adblocker to something like: 

"Die Option zur Deaktivierung der Analyse ihrer Aktionen (Tracking) kann nicht angezeigt werden, da es von ihrem Browser blockiert wird. Möglicherweise blockiert ihr Browser ohnehin das Tracking, sodass Sie Ihr Einverständnis dafür hier nicht mehr entziehen müssen. Wenn Sie auf Nummer sicher gehen wollen, sollten Sie ihren AdBlocker deaktivieren und die Seite neu laden, um manuell die Erlaubnis zum Tracken zu entziehen."

Highlight this text by making it fat or in another color.

File: app/package.json [hunk 1]
```diff
@@ -4,8 +4,9 @@
   "private": true,
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint --max-warnings 0 --no-fix",
+    "build:production": "vue-cli-service build --mode production",
+    "build:staging": "vue-cli-service build --mode staging",
+    "lint": "vue-cli-service lint",
```
Comment (xcaverxiii) [inline app/package.json:9]: the lint --max-warnings 0 --no fix got lost

File: app/src/views/Privacy.vue [hunk 2]
```diff
@@ -245,6 +245,53 @@
...
+          <li>URL der vorhergehenden Seite (sofern diese das erlaubt)</li>
+          <li>Bildschirmauflösung</li>
+          <li>Lokale Zeit</li>
+          <li>Externe Links</li>
+          <li>Dauer des Seitenaufbaus</li>
+          <li>User Agent des Browsers</li>
+        </ul>
+
+        <br />
+        <p v-if="!optOutIframeExists()">
```
Comment (xcaverxiii) [inline app/src/views/Privacy.vue:282]: dont use a function here, use a data attribute which is set by mounted or some other lifecycle hook.

**Review by xcaverxiii — CHANGES_REQUESTED** [review]

**Comment by theErikss0n** [comment]
The information, that adblock can block the i-frame is displayed all the time now.
It would not be worth the effort for now to only display it, if the i-frame is blocked.
It is not easy to implement it this way

**Review by DianaPankratz — CHANGES_REQUESTED** [review]
Please add a line break after the text. Currently in Opera it looks like this: 
image

**Review by DianaPankratz — APPROVED** [review]

**Review by xcaverxiii — APPROVED** [review]

**Pull request merged**